src/issue/screens/issue.screen.js

```diff
@@ -202,6 +208,27 @@ class Issue extends Component {
     this.commentsList.scrollToEnd();
   };
 
+  deleteComment = comment => {
+    const { repository } = this.props;
+    const { id } = comment;
+
+    const repoName = repository.name;
+    const owner = repository.owner.login;
+
+    this.props.deleteIssueCommentByDispatch(id, owner, repoName);
```

Comment:
I do not know for sure, but it makes sense to define the variable `id`, if it will be used once, I mean what maybe just pass `comment.id`?

Reply:
yeah, you are completely right! 😄